README.md

```diff
@@ -41,7 +41,7 @@ Depending on your local platform, you'll need to add the correct version of [ful
 
 ### Dev
 
-You'll need to start by installing the packages. At the root, run `yarn install` (or `yarn` if you're cool like that). Because this wallet has native packages, you'll need to drop into the app/ directory `cd app/` and run `yarn install` here as well. Once done, hop back up to the root directory (I know you know this, but `cd ..`).
+You'll need to start by installing the packages. At the root, run `yarn install`. Because this wallet has native packages, you'll need to drop into the app/ directory `cd app/` and run `yarn install` here as well. Once done, hop back up to the root directory (I know you know this, but `cd ..`).
```

Comment:
I'd say it should be "and run 'yarn install' THERE as well" -- substitute THERE for HERE.

Reply:
I guess it depends if we assume the user is still in the root dir, or has moved into the app/ dir... if we're in the app/ dir then probably "here" is fine.

Reply:
Nope, still wrong: it's  "go somewhere and THERE do something"